example_app/app.ts

```diff
@@ -0,0 +1,43 @@
+import { Drash, SocketServer } from "./deps.ts";
+
+import HomeResource from "./src/resources/home_resource.ts";
+import ChatResource from "./src/resources/chat_resource.ts";
+
+const webServer = new Drash.Http.Server({
+  response_output: "application/json",
+  directory: Deno.realPathSync("."),
+  static_paths: ["/public"],
+  resources: [
+    HomeResource,
+    ChatResource,
+  ],
+  template_engine: true,
+  views_path: "./src/views",
+});
+
+webServer.run({
+  hostname: "localhost",
+  port: 3001,
+});
+console.log(`Web server started on ${webServer.hostname}:${webServer.port}`);
+
+const socketServer = new SocketServer();
+socketServer.run({
+  hostname: "localhost",
+  port: 3000,
+});
+console.log(
+  `Socket server started on ${socketServer.hostname}:${socketServer.port}`,
+);
+
+socketServer
+  .createChannel("Channel 1")
+  .onMessage((incomingEvent: any) => {
```

Comment:
`incomingMessage`?

Reply:
i like that better. done. d800f624f78496bcc8d28487b32b5f875a88b99e